script/install

```diff
@@ -80,6 +80,7 @@ curl -LSs --output /tmp/tuistenv.zip https://storage.googleapis.com/tuist-releas
 ohai "Unzipping tuistenv..."
 unzip -o /tmp/tuistenv.zip -d /tmp/tuistenv > /dev/null
 ohai "Installing tuistenv..."
+[ ! -d "/usr/local/bin" ] && execute_sudo mkdir /usr/local/bin/
```

Comment:
Nit: maybe using `if condition; then ... fi` might be more readable (and it's used in the shell script above?)

Reply:
I agree, it makes it more readable. I added that change